File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/commands/Orders/ConfirmPayment.cs
using System;
using ENode.Commanding;

namespace Registration.Commands.Orders
{
    public class ConfirmPayment : Command<Guid>
    {
        public bool IsPaymentSuccess { get; set; }

        public ConfirmPayment() { }
        public ConfirmPayment(Guid orderId, bool isPaymentSuccess) : base(orderId)
        {
            IsPaymentSuccess = isPaymentSuccess;
        }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/Orders/ConfirmPayment.java
package com.microsoft.conference.common.registration.commands.Orders;

import org.enodeframework.commanding.Command;

public class ConfirmPayment extends Command<String> {
    public boolean IsPaymentSuccess;

    public ConfirmPayment() {
    }

    public ConfirmPayment(String orderId, boolean isPaymentSuccess) {
        super(orderId);
        IsPaymentSuccess = isPaymentSuccess;
    }
}
